.github/workflows/integ_test_api.yml

```diff
@@ -83,7 +83,6 @@ jobs:
           scheme: AWSAPIPluginGraphQLUserPoolTests
 
   api-graphql-iam-test:
-    if: ${{ false }}
```

Comment:
typo on re-enabling the test, should be `api-rest-iam-test` .

Additional context:
`api-graphql-iam-test` should be re-enabled as well but I disabled it because it is failing in the Github workflows. This test runs fine locally on my backend. I think the CI backend needs an re-provision to see why if that's the problem (tests and provisioning steps changed, but backend not updated?). IAM interceptor is used for GraphQL APIs as well so it would make sense to enable this as well, but I tested this one locally and it is passing, so we can do this change in a subsequent PR

Reply:
Thanks, silly miss on my part. Addressed.